src/main/java/testUI/TestUIServer.java

```diff
@@ -289,11 +289,14 @@ public static void stop(int driver) {
             removeUsePort(driver - 1);
             removeUseBootstrapPort(driver - 1);
             if (Configuration.automationType.equals(IOS_PLATFORM)) {
-                getDrivers().get(driver - 1).close();
+                if (getDrivers().size() != 0)
+                    getDrivers().get(driver - 1).close();
                 sleep(500);
             }
-            getDrivers().get(driver - 1).quit();
-            removeDriver(driver - 1);
+            if (getDrivers().size() != 0) {
+                getDrivers().get(driver - 1).quit();
+                removeDriver(driver - 1);
+            }
```

Comment:
This looks sus. If iOS platform is used then driver can get deleted twice. Also, please wrap `if` statements in curly braces for readability